lib/vector2/CommonVectors.dm

```diff
@@ -2,7 +2,7 @@
 	Make sure you don't modify these vectors.
 	You should treat all vectors as immutable in general.
 */
-var Vector2/Vector2 = new
+var/global/Vector2/Vector2 = new
```

Comment:
var/global bad idea. Use GLOBAL_REAL

Reply:
this is used quite a lot as its a holder for functions, having to put GLOB infront of it would be a pain

Reply:
> this is used quite a lot as its a holder for functions, having to put GLOB infront of it would be a pain

GLOBAL_REAL don't need use GLOB. Check define GLOBAL_REAL

Reply:
if you could make a direct suggestion as to how I should write this line that would be great.

Reply:
> if you could make a direct suggestion as to how I should write this line that would be great.

GLOBAL_REAL(Vector2, /Vector2) = new